### ISSUE TITLE
[BUG] Dual state buttons losing state on board change

### ISSUE THREAD

**Issue by alvarovazquez** [rivafarabi/deckboard#564]
**Describe the bug**
When a dual state button is on it's active state, if you change to another board (swiping or through a button action) and go back to the button's board, the button is on it's inactive state even if the StreamLabs OBS scene is active.

**To Reproduce**
1. Create two boards, one with a dual state button to change a Streamlabs OBS scene.
2. Tap the button and make sure the scene is selected and the button chantes to it's active state
3. Change to the second board (swiping)
4. Go back to the first board
5. The button is no longher on it's active state

**Expected behavior**
The button should keep it's active state.

**Server App (please complete the following information):**
 - OS: [Windows 10]
 - Version 10.0.19041 build 19041

**Client App (please complete the following information):**
 - OS: Android 9 (Xiaomi Mi A1 with Nova Launcher)
 - Version 9

**Additional context**
The Android device is connected through WiFi to the Windows server with no problems that I can tell.

**Comment by JustMazey** [comment]
That's the exact same thing that I wrote but you wrote it more understandable.
You have to do another action like toggle a source or change scene to get it to know what's active again.

**Comment by alvarovazquez** [comment]
Yes, I have just seen that you posted it as a Feature Request: https://github.com/rivafarabi/deckboard/issues/561

I looked among the bugs right before creating this report in order not to duplicate issues, that's why I didn't find it.

Anyway, I think it's OK to have separate reports to help the author managing them.

Keep up the great work! I already purchased the PRO version of the Android App 😃
